main.go

```diff
@@ -317,14 +318,16 @@ func mustLoadRouter(logger *zap.Logger, config *Config, indexer Indexer) *mux.Ro
 }
 
 func getRouter(logger *zap.Logger, config *Config, indexer Indexer) (*mux.Router, error) {
+	if !featureProxyMode {
```

Comment:
Should it be the other way around? :thinking: 
```suggestion
	if featureProxyMode {
```

Reply:
Right, fixed